tools/azure-sdk-tools/packaging_tools/generate_client.py

```diff
@@ -2,11 +2,31 @@
 import logging
 from pathlib import Path
 from subprocess import run
-import sys
+import tomli
 
 _LOGGER = logging.getLogger(__name__)
 
 
+CONF_NAME = "pyproject.toml"
+
+def check_post_process(folder: Path) -> bool:
+    conf_path = folder / CONF_NAME
+    if not conf_path.exists():
+        return False
+
+    with open(conf_path, "rb") as fd:
+        toml_dict = tomli.load(fd)
+        if toml_dict.get("tox-generate", None) is not None:
+            return toml_dict["tox-generate"]["autorest-post-process"]
```

Comment:
I think we need to adjust how we're accessing the dict. If I read the pyproject.toml from, for example, textanalytics, I get a dict that looks like this:

```
>>> import tomli
>>> with open("pyproject.toml", "rb") as fd:
...     toml_dict = tomli.load(fd)
...
>>> toml_dict
{'tool': {'azure-sdk-build': {'pyright': False}}}
```

We have a section for azure-sdk-build checks, do we want to add one for generation options?

Reply:
something like `[tools.generate]`? I think Laurent originally suggested that, I'm good updating to use that

Reply:
Changed it so the section is `[tool.generate]`